Makefile

```diff
@@ -5,7 +5,7 @@ OTELCOL_BUILDER_VERSION ?= 0.109.0
 OTELCOL_BUILDER_DIR ?= ${HOME}/bin
 OTELCOL_BUILDER ?= ${OTELCOL_BUILDER_DIR}/ocb
 
-DISTRIBUTIONS ?= "otelcol,otelcol-contrib,otelcol-k8s"
+DISTRIBUTIONS ?= "otelcol,otelcol-contrib,otelcol-k8s,otelcol-otlp"
 GEN_CONFIG_DISTRIBUTIONS ?= "otelcol,otelcol-contrib"
```

Comment:
This distribution's goreleaser file should be generated. Unlike the k8s distro that has unique artifact requirements, this distribution's artifacts should match contrib and core.

Reply:
The problem is that the generated goreleaser file expects a config file to install. If we want to stop including a default config, we would have to make the generator much more flexible. Should I work on that in this PR?

Reply:
If we need to do this, I vote for doing the refactor in a separate PR that we merge before this one

Reply:
Ok, we'll move forward without generating the goreleaser file.